openslides/motion/templates/motion/slide.html

```diff
@@ -77,6 +77,9 @@ <h4 class="first">{% trans "Status" %}:</h4>
     {{ motion.active_version.title }}
     <small>
         {% trans "Motion" %} {{ motion.identifier|default:'' }}
+        {% if motion.is_amendment %}
+            (Amendment of <strong>{{ motion.parent }}</strong>)
```

Comment:
I prefer that we reference a motion the same way at any place. @normanjaeckel what do you think?